src/item.js

```diff
@@ -42,7 +42,7 @@ module.exports = function(itemConfig, index) {
   function getData() {
     switch (_type) {
       case CONSTANTS.itemType.DATA:
-        return _config;
+        return omit(_config, 'eventInfo');
```

Comment:
I'd use a constant:
return omit(_config, Object.keys(ITEM_CONSTRAINTS.event.eventInfo));

Reply:
`return omit(_config, Object.keys(ITEM_CONSTRAINTS.event.eventInfo));` will not return `eventInfo` key. We could use `return omit(_config, Object.keys(ITEM_CONSTRAINTS.event));` then we omit both `event` and `eventInfo` keys which I think is good anyway. We should not allow pushing events as `DATA` type. WDYT @jckautzmann ?

Reply:
getData() should only return data, no event. So this makes sense:
`return omit(_config, Object.keys(ITEM_CONSTRAINTS.event));`